### PR TITLE
Updated downloads.html to include 2.7.1

### PR DESCRIPTION
**Pull request by mimaison**

### CODE REVIEW

**Comment by mimaison** [comment]
@mjsax @vvcephei can you take a look? Thanks

File: downloads.html [hunk 1]
```diff
@@ -60,23 +60,66 @@ <h3 class="download-version">2.8.0<a href="#2.8.0"><i class="fas fa-link " style
...
+                <li>Scala 2.12 &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/2.7.1/kafka_2.12-2.7.1.tgz">kafka_2.12-2.7.1.tgz</a> (<a href="https://downloads.apache.org/kafka/2.7.1/kafka_2.12-2.7.1.tgz.asc">asc</a>, <a href="https://downloads.apache.org/kafka/2.7.1/kafka_2.12-2.7.1.tgz.sha512">sha512</a>)</li>
+                <li>Scala 2.13 &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/2.7.1/kafka_2.13-2.7.1.tgz">kafka_2.13-2.7.1.tgz</a> (<a href="https://downloads.apache.org/kafka/2.7.1/kafka_2.13-2.7.1.tgz.asc">asc</a>, <a href="https://downloads.apache.org/kafka/2.7.1/kafka_2.13-2.7.1.tgz.sha512">sha512</a>)</li>
+            </ul>
+            We build for multiple versions of Scala. This only matters if you are using Scala and you want a version
+            built for the same Scala version you use. Otherwise any version should work (2.13 is recommended).
+        </li>
+    </ul>
+
+    <p>
+        Kafka 2.7.1 includes a number of significant new features. Here is a summary of some notable changes:
```
Comment (mjsax) [inline downloads.html]: Seems this list was copied from `2.7.0` release below, but does not really apply to a bug fix release? I think this paragraph and the bullet point list below should not be added, but only the last paragraph about `2.7.1`.

**Comment by mimaison** [comment]
@mjsax Can you take another look? Thanks

**Comment by mimaison** [comment]
@rajinisivaram @omkreddy Can you take a look? We need this PR and https://github.com/apache/kafka/pull/10660 in order to release 2.7.1. Thanks

**Review by omkreddy — APPROVED** [review]
@mimaison Thanks for the PR. LGTM

**Pull request merged**